UnityProject/Assets/Scripts/Weapons/Projectiles/Behaviours/ProjectileDamageLivingHealth.cs

```diff
@@ -28,9 +30,19 @@ public bool OnHit(RaycastHit2D hit)
 
 		private bool TryDamage(RaycastHit2D hit)
 		{
+			var newDamage = damageData.Damage;
 			var coll = hit.collider;
 			var livingHealth = coll.GetComponent<LivingHealthBehaviour>();
 			if (livingHealth == null) return false;
+			// checks if its a kinetic weapon is a high atmosphere
+			if (damageData.DamageType== DamageType.Kinetic)
+			{
+				
+				if(MatrixManager.AtPoint((Vector3Int)hit.point.To2Int(), true).MetaDataLayer.Get(hit.transform.localPosition.RoundToInt()).GasMix.Pressure <= 50)
+				{
+					newDamage=damageData.Damage * .25f;
+				}
+			}
 
 			livingHealth.ApplyDamageToBodypart(shooter, damageData.Damage, damageData.AttackType, damageData.DamageType, targetZone);
```

Comment:
Think that needs newDamage and not damageData.Damage for the ApplyDamageToBodyPart

Reply:
yes, that's asking the livingHealth to do the damage before reduction